test/Microsoft.Azure.Devices.Edge.Test.Common/linux/EdgeDaemon.cs

```diff
@@ -35,47 +35,14 @@ public async Task InstallAsync(Option<string> packagesPath, Option<Uri> proxy, C
                     properties = properties.Append(p).ToArray();
                 });
 
-            string[] commands = await packagesPath.Match(
-                p =>
+            string[] commands = packagesPath.Match(
+                (p) =>
```

Comment:
Nit: you only need parenthesis if you don't have any arguments

Reply:
got it